### PR TITLE
fix: [CDS-53034]: Added missing deployment freeze resource type and module icon in Audit trails

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

https://user-images.githubusercontent.com/98325173/217209896-5fac9e58-5b10-47eb-85bc-a9ad10d01e76.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/72-freeze-windows/RouteDestinations.tsx [hunk 1]
```diff
@@ -32,6 +34,27 @@ RbacFactory.registerResourceTypeHandler(ResourceType.DEPLOYMENTFREEZE, {
   }
 })
 
+AuditTrailFactory.registerResourceHandler('DEPLOYMENT_FREEZE', {
```
Comment (ishantmehtaharness) [inline src/modules/72-freeze-windows/RouteDestinations.tsx:37]: Can we use `ResourceType.DEPLOYMENT_FREEZE` here?

Comment (Sagar-Rajput7) [reply src/modules/72-freeze-windows/RouteDestinations.tsx:37]: @ishantmehtaharness  the resource type coming from DTO is different from the one defined in ResourceType i.e. `DEPLOYMENTFREEZE`

Comment (Sagar-Rajput7) [reply src/modules/72-freeze-windows/RouteDestinations.tsx:37]: Also, this prop only accept `resourceType: ResourceType` this, so we are safe on that.

Comment (Sagar-Rajput7) [reply src/modules/72-freeze-windows/RouteDestinations.tsx:37]: Also, this prop only accepts, `ResourceDTO['type']`, so we are safe on that.

**Review by ishantmehtaharness — APPROVED** [review]
Looks GOod

**Review by Eric761 — APPROVED** [review]

**Pull request merged**